src/components/buttonlist/buttonlist.js

```diff
@@ -275,62 +276,68 @@ var ButtonList = Component.extend({
   /*
    * reset buttons show state
    */
-  _showAllButtons: function() {
-    var _this = this;
-    // show all existing buttons
-    var buttons = this.element.selectAll(".vzb-buttonlist-btn");
-    buttons.each(function(d,i) {
-      var button = d3.select(this);
-      if (_this.getLayoutProfile() !== 'small') {
-        if (button.style("display") === "none"){
-          button.style("display", "inline-block");
-        }
-      } else {
-        button.style("display", '');
-      }
-    });
-  },
+   _showAllButtons: function() {
+     // show all existing buttons
+     var _this = this;
+     var buttons = this.element.selectAll(".vzb-buttonlist-btn");
+     buttons.each(function(d,i) {
+       var button = d3.select(this);
+       if (button.style("display") == "none"){
+         if(d.id != "trails" && d.id != "lock"){
```

Comment:
Here I need to show all buttons, it is some kind of reset. But these two are the exception because they behave differently, they will only show if we have space and if there is an entity selected.